interview-questions.md

```diff
@@ -7,53 +7,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: Most Ruby methods are accessors. To turn them into mutators, we can use the ! operator. Another option would be to declare a variable and assign it to the variable.method.  
 
-  Researched answer:
+  Researched answer: As mentioned above, most Ruby methods are accessors and can be made permanent with the ! operator. The methods with ! are specifically called bang methods. If the bang method is executed and did not mutate the object, it will return nil. I was also reading that using bang methods aren't ideal because they're considered destructive. 
+  
+  Question for Austin: Does (most Ruby methods being accessors) have anything to do with the nature of Ruby and instance variables? I.e. naturally if you pass an argument(i.e. maybe the variable we want to use a method on?) into a class, it is considered local scope and we don't really affect the argument passed in. 
```

Comment:
More or less: OOP doesn't do a whole lot of changing the data it receives. It makes new data built from the data it receives and passes that along so as to not corrupt the stream of information being passed around. I am not sure if this is naturally emergent of OOP design or if it was a considered choice when building the framework.